### DISCUSSION TITLE
[Scam Alert] ICBroker

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#33] [Announcements]
<div dir="rtl">

**Project Name:** 
ICBroker

**نام پروژه:** 
ICBroker

**نماد پروژه:**

**اتفاقی که افتاده است:**
صفحه
legal documents
در سایت خودشون کاملا خالیه

![photo_2021-09-10_23-48-37](https://user-images.githubusercontent.com/87842536/132906085-22c09b8f-dcad-4d1d-964d-313051ec8109.jpg)

این درواقع هشداری هست که خود سایت ایسی بروکر گذاشته:

CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. The vast majority of retail client accounts lose money when trading in CFDs. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money. There is a possibility to lose all your initial capital. Before trading you should fully understand the true extend of your exposure to the risk of loss and your level of experience. Please note that pass performance does not constitute a reliable indicator of future results. Future forecasts do not constitute a reliable indicator of future performance. If you do not fully understand the risks involved and before deciding to trade, you should carefully consider your investment objectives, level of experience and risk tolerance, you should become aware of all the related risks and seek independent advice and suitability licensed financial advisor. For further information please consider our
Risk Disclosure Notice.
CFD

**قوانینی که نقض شده است:**

**نوع هشدار:**
هشدار کلاهبرداری

**نوع کلاهبرداری:** 
طرح پانزی، طرح هرمی

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
https://www.icbroker.com/
آدرس کامیونیتی فارسی در اینستاگرام
https://www.instagram.com/icbroker_ir/

**برآورد مبلغ کلاهبرداری:**

**منابع:**

**جمع بندی:**

**Comment by saeedkousha** [comment]
CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. The vast majority of retail client accounts lose money when trading in CFDs. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money. There is a possibility to lose all your initial capital. Before trading you should fully understand the true extend of your exposure to the risk of loss and your level of experience. Please note that pass performance does not constitute a reliable indicator of future results. Future forecasts do not constitute a reliable indicator of future performance. If you do not fully understand the risks involved and before deciding to trade, you should carefully consider your investment objectives, level of experience and risk tolerance, you should become aware of all the related risks and seek independent advice and suitability licensed financial advisor. For further information please consider our
Risk Disclosure Notice.
CFD

این درواقع هشداری هست که خود سایت ایسی بروکر گذاشته

**Reply by saeedkousha** [reply]
نکته بعدی اینکه صفحه 
legal documents
در سایت خودشون کاملا خالیه

**Comment by saeedkousha** [comment]
آدرس کامیونیتی فارسی در اینستاگرام
icbroker_ir

**Comment by mehran-mrn** [comment]
مشاهده اين لينك ها مفيد خواهد بود

[https://www.aparat.com/v/AMSC0](https://www.aparat.com/v/AMSC0)

[https://www.scamadviser.com/check-website/icbroker.com](https://www.scamadviser.com/check-website/icbroker.com)

[https://fakewebsitebuster.com/fxtdpro-com/](https://fakewebsitebuster.com/fxtdpro-com/)

[https://webamoozcom.github.io/warning-list/](https://webamoozcom.github.io/warning-list/)

[http://icbroker.blogfa.com/](http://icbroker.blogfa.com/)
